app/flatpak-builtins-info.c

```diff
@@ -224,6 +226,7 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
       len = MAX (len, strlen (_("Arch:")));
       len = MAX (len, strlen (_("Branch:")));
       len = MAX (len, strlen (_("Version:")));
+      len = MAX (len, strlen (_("License:")));
```

Comment:
Seems we're missing an `if (appdata_license != NULL)` here (and similarly for appdata_version but perhaps that should be a separate commit)